package.json

```diff
@@ -2,15 +2,30 @@
   "name": "vote.gov",
   "version": "1.0.0",
   "main": "gulpfile.js",
+  "config": {
+    "hugo": {
+      "en": "config.toml",
+      "es": "config_es.toml"
+    },
+    "urls": {
+      "en": {
+        "staging": "https://vote-gov-staging.apps.cloud.gov/",
+        "production": "https://vote-gov.apps.cloud.gov/"
+      },
+      "es": {
+        "staging": "https://vote-gov-staging.apps.cloud.gov/es/",
+        "production": "https://vote-gov.apps.cloud.gov/es/"
+      }
+    }
+  },
```

Comment:
Hey @yozlet,

So I found some better documentation than what I linked to in the opening comment [around `npm config` here](https://docs.npmjs.com/misc/config#per-package-config-settings). They're using the custom config variables for `{ "port": "8080" }`.

I love the idea of scoping these further down like in solution `1.`. I had the deployment config in `circle.yml` originally, but there was no transparency and no easy way to update the variables without looking into CircleCI's project settings page and documenting how to upload it there for other team members. So I figured the `package.json` was good enough. As far as adding a `.env` package, I think that would be good but it would add the headache of needing to have documentation and everyone on the project using the `dotenv` Ruby gem for the project.

I really like your idea of using the `env` variables in the gulp tasks, though. _I believe that I can do just that by leveraging `npm run *` commands_. For now though, I think we should leverage what we have in this PR and move forward with the proposed changes of putting all the custom config under a single key. Another PR can add using `env` variables with gulp tasks as there is still some more work on deployments and building the multi-lingual site.